crypto/params.c

```diff
@@ -12,6 +12,26 @@
 #include <openssl/params.h>
 #include "internal/thread_once.h"
 
+OSSL_PARAM *OSSL_PARAM_locate2(OSSL_PARAM *p, const char *key, int found[])
```

Comment:
I'd much prefer there to only be one locate call.  Adding a second with an optional argument isn't good.  Especially since these are brand new APIs.

Reply:
At worst, a macro with two arguments invoking the function with three.